playbooks/roles/caasp4/templates/inventory-caasp4.yml.j2

```diff
@@ -32,15 +32,15 @@ airship-openstack-control-workers:
     ansible_python_interpreter: /usr/bin/python3
 
 airship-ucp-workers:
-  hosts: *firsttwoworkers
+  hosts: *first_workers
   vars:
     ansible_user: sles
     ansible_python_interpreter: /usr/bin/python3
 
 airship-openstack-compute-workers:
   hosts:
 {% for worker_ip in _terraform_json_output.ip_workers.value %}
-{% if loop.index0 > 1 %}
+{% if loop.index0 >= airship_ucp_control_workers %}
```

Comment:
urgs. so we (again) include $random variables from everywhere in specific roles. But not your fault..